File: Dsa/common/arraySort.js
function ArraySort(arr){
    let n= arr.length
    for(i=0; i<n; i++){
        for(j=0; j<n-1-i; j++){
            if(arr[j]> arr[j+1]){
                let temp=arr[j]
                arr[j]=arr[j+1]
                arr[j+1]=temp
            }
        }
    }
    return arr
}
console.log(ArraySort([2,5,7,1,4,3]))

let arr=[5,6,7,4,3,9,2]

const accending=arr.slice().sort((a,b)=> a-b)
console.log(accending)

File: Dsa/common/arraySort.py
def ArraySort(arr):
    n=len(arr)
    for i in range(n):
        for j in range(n-1-i):
            if arr[j]>arr[j+1]:
                arr[j], arr[j+1]=arr[j+1], arr[j]
    return arr

print(ArraySort([6,7,8,9,4,2,1]))

print(sorted([5,7,6,4,3,1,2]))